### DISCUSSION TITLE
Thinking about to make a new logo for uBlock Origin...

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#1854] [Show and tell]
Funny to imagine there's only three things they have in common.

**Comment by gorhill** [comment]
Thanks for the offer, but I rather not change the icon.

**Comment by u-RraaLL** [comment]
OP edited the title and the message before deleting their account...

I'm not sure if a previous version of the post can be brought back, but the post remains visible in the edit history, so maybe just restoring the discussion name would suffice?
